src/main/webapp/assets/style.css

```diff
@@ -22,6 +23,10 @@ body {
   padding: 0;
 }
 
+.title {
+  padding: 0 0 0 10rem;
+}
+
```

Comment:
On the desktop version, this makes the title of the page line up with the map but on the mobile version, it looks like an off indent. I think you were trying to line it up with the map as well for the mobile version so that could be added as a `@media screen` in the CSS!

Reply:
Good catch, I'll make adjustments!